### DISCUSSION TITLE
Pls Add A feature that allows to download images

### DISCUSSION THREAD

**Discussion by TheAnay** [Tyrrrz/DiscordChatExporter#733] [General]
Pls Add A feature that allows to download images. It would be very thankfull

**Comment by 96-LB** [comment]
![image](https://user-images.githubusercontent.com/21089343/145307131-f048be17-3630-42fd-a270-bf697c7d8a72.png)
![image](https://user-images.githubusercontent.com/21089343/145307162-22fc448b-1cda-4921-85e2-9ade35ac6476.png)
***
Or, if you're using CLI, use the `--media` flag.
